server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/http/Request.java

```diff
@@ -0,0 +1,18 @@
+package com.amazonaws.ml.mms.servingsdk.http;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Request {
```

Comment:
Javadoc

Reply:
I will add javadoc and license headers  in the next iteration. I am not publishing this yet.